src/App.vue

```diff
@@ -33,6 +33,7 @@ export default {
   },
   data() {
     return {
+      vueCliMode: process.env.NODE_ENV,
```

Comment:
I tried referencing NODE_ENV right in the template but it comes back as undefined. Not sure if it's possible to use it outside of `script` tags.